### DISCUSSION TITLE
Option to disable entity teleportation with the end_portal safe-teleporting feature

### DISCUSSION THREAD

**Discussion by LeeTheTech** [PurpurMC/Purpur#968] [__Completed__] [answered]
Hello, I was hoping you could add a toggle in the config for this, the reason being is entities are teleported to the spawn and it's annoying when a dragon egg or a boat gets teleported to spawn because it can't be broken without a admin.
![2022-05-03_15 41 02](https://user-images.githubusercontent.com/8154217/166584728-5f059cc4-7c83-41a3-9c3d-3b8829e6ceb3.png)
![2022-05-03_15 40 21](https://user-images.githubusercontent.com/8154217/166584731-c02ef777-f21b-481c-ac8a-fcd12dff1081.png)
.

**Comment by encode42** [comment]
The `safe-teleporting` option is actually related to a fix to gravity block duplication!

However, we should still look into implementing this feature, considering we have the [entities-can-use-portals](https://purpurmc.org/docs/Configuration/#entities-can-use-portals) option.

**Comment by encode42** [comment, marked as answer]
Just checked, and luckily for you, this is already implemented!

Just disable [entities-can-use-portals](https://purpurmc.org/docs/Configuration/#entities-can-use-portals) in `purpur.yml`, and your entities will not use portals.

**Reply by LeeTheTech** [reply]
Would that include the nether portal? If so could there be toggles for each type of portal?
